lib/spack/docs/packaging_guide.rst

```diff
@@ -2016,45 +2016,133 @@ the Python extensions provided by them: once for ``+python`` and once
 for ``~python``.  Other than using a little extra disk space, that
 solution has no serious problems.
 
------------------------------------
-Implementing the ``install`` method
------------------------------------
+.. _installation_procedure:
 
-The last element of a package is its ``install()`` method.  This is
+---------------------------------------
+Implementing the installation procedure
```

Comment:
I see -- I'm still thinking on it.  "Build pipeline" is the best I can think of so far.  Let's just leave it as-is.  I agree it's not just `install()` anymore.